util/validators.js

```diff
@@ -264,6 +264,53 @@ module.exports.validateCreateTaskInput = (
   };
 };
 
+module.exports.validateCreateResourceInput = (
+  title,
+  link,
+  description,
+  image,
+  podcast
+) => {
+
+  const errors = {};
+
+  const titleValidator = /^[a-zA-Z0-9- ]{6,50}$/i;
+
+  const linkValidator = /^https?:\/\/([0-9\x41-\x5A\x61-\x7A\xC0-\xF6\xF8-\xFF.-]){1,63}(\.)[0-9\x41-\x5A\x61-\x7A\xC0-\xF6\xF8-\xFF0-9]{1,63}(((\/)[0-9\x41-\x5A\x61-\x7A\xC0-\xF6\xF8-\xFF-=?#&]{1,63}){1,63})?(\/)?((\.)[0-9\x41-\x5A\x61-\x7A\xC0-\xF6\xF8-\xFF.-=?#&]{1,63})?$/gm;
+
+  //Other link valiadators, will take out later
+  //Without other countries characters: /^https?:\/\/([a-zA-Z0-9.-]){1,63}(\.)[a-zA-Z0-9]{1,63}(((\/)[a-zA-Z0-9-]{1,63}){1,63})?(\/)?((\.)[a-zA-Z0-9.]{1,63})?$/gm;
+  //With other chars in domain name /^https?:\/\/([0-9\x41-\x5A\x61-\x7A\xC0-\xF6\xF8-\xFF.-]){1,63}(\.)[a-zA-Z0-9]{1,63}(((\/)[a-zA-Z0-9-=?#&]{1,63}){1,63})?(\/)?((\.)[a-zA-Z0-9.-=?#&]{1,63})?$/gm
+
+  if (title.trim() === "") {
```

Comment:
try to use truthy/falsey values if possible. ie instead of
`if(title.trim() === "")`
try
`if(!title.trim())